openstack_virtual_baremetal/deploy.py

```diff
@@ -198,16 +214,17 @@ def _deploy(stack_name, stack_template, env_paths, poll):
                           files=all_files,
                           parameters=parameters)
 
-    print 'Deployment of stack "%s" started.' % stack_name
+    print('Deployment of stack "%s" started.' % stack_name)
     if poll:
         _poll_stack(stack_name, hclient)
 
-def _poll_stack(stack_name,  hclient):
+
+def _poll_stack(stack_name, hclient):
     """Poll status for stack_name until it completes or fails"""
-    print 'Waiting for stack to complete',
+    print('Waiting for stack to complete')
```

Comment:
This isn't equivalent to the previous statement.  It actually needs to be

print('Waiting for stack to complete', end='')

Which for Python 2 support will also need

from __future__ import print_function

at the top of the file.